### ISSUE TITLE
Sane repository description

### ISSUE THREAD

**Issue by abitrolly** [facebookincubator/fastmod#47]
Add "find and replace tool" into repository description. Otherwise there is absolutely no way for search algorithms to discover the tool. And add some topics there like https://github.com/topics/find-and-replace https://github.com/topics/find and https://github.com/topics/replace

**Comment by nk9** [comment]
I would agree, most people don't know what `codemod` is. Far more people could use this tool than presently have heard of it. I think I learned of it via ChatGPT, but most people are going to be searching for some key words.

A more search-friendly repo description could be: "Search and replace regex strings across many files interactively." This description is even more important on package managers like [Homebrew](https://formulae.brew.sh/formula/fastmod). There, the same inscrutable description exists, but without any of the README context—and there isn't even a `codemod` formula to give you a clue.

**Comment by swolchok** [comment]
Updated the description, partly based on the codemod tool's description. What do you think?

**Comment by abitrolly** [comment]
@swolchok I find references for `codemod` most confusing. From what I see `fastmod` is a standalone tool, not some "fast partial replacement" add-on for `codemod`.

For the description it would be sufficient to say that "`fastmod` is interactive bulk search/replace tool" and then expand in README that it is based on `codemod` and explain differences between two.

Personally, I would explain `fastmod` as a "missing replace for `ripgrep`". And add `terminal` to the list of repo topics.

**Comment by abitrolly** [comment]
And the `codemod` is dead - https://github.com/facebookarchive/codemod

And the reference to is is even more confusing, because there is now https://github.com/codemod/codemod with 1k stars (against 4k used by the old one).


"Interactive find/search and replace in terminal".

This is how I would call it. If people need the tool, then can follow URL and read more about `codemod ` history,
